main.py

```diff
@@ -0,0 +1 @@
+print("hello world")
```

Comment:
Hi! Welcome to the course!
All work concerning the first lab should be located in the `lab_1_classify_by_unigrams` folder.
Please, remove extra files.

Reply:
This file still exists. Please, remove it.